src/interactions/userContext/resetTag.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  CacheType,
+  CommandInteraction,
+  GuildMember,
+  PermissionsBitField,
+  UserContextMenuCommandInteraction,
+} from "discord.js";
+import {
+  CustomInteractionReplyOptions,
+  InteractionCommand,
+} from "../../classes/CustomInteraction";
+
+import { ApplicationCommandType } from "discord-api-types/v10";
+import { CustomClient } from "lib/client";
+
+export default class ContextCommand extends InteractionCommand {
+  /**
+   *
+   */
+  constructor(client: CustomClient) {
+    super(client, {
+      type: ApplicationCommandType.User,
+      name: "Reset To Tag",
+      defaultMemberPermissions: new PermissionsBitField("Administrator"),
+    });
+  }
+
+  async execute(
+    interaction: CommandInteraction<CacheType>
+  ): Promise<CustomInteractionReplyOptions> {
+    const int = interaction as UserContextMenuCommandInteraction;
+
+    if (!(int.targetMember instanceof GuildMember)) {
+      return { content: "Resetting to tag only works on guild members", eph: true };
+    }
+    if (this.client.permlevel(undefined, int.targetMember) >= 2) {
+      return { content: "Helper and above cannot be nicknamed", eph: true };
+    }
+    if (int.targetUser.discriminator !== "0") {
+      return { content: `Use "Reset Name" instead `, eph: true };
+    }
+    await this.client.setMemberName(int.targetMember, int.targetUser.tag);
```

Comment:
```suggestion
    await this.client.setMemberName(int.targetMember, int.targetUser.username);
```
If we use .tag, it'll add the #0, which we don't want.